tests/auxil/bot_method_checks.py

```diff
@@ -315,6 +315,7 @@ async def check_defaults_handling(
     defaults_no_custom_defaults = Defaults()
     kwargs = {kwarg: "custom_default" for kwarg in inspect.signature(Defaults).parameters}
     kwargs["tzinfo"] = pytz.timezone("America/New_York")
+    kwargs.pop("disable_web_page_preview")  # mutually exclusive with link_preview_options
```

Comment:
@Bibo-Joshi is this an acceptable way to pass the `check_defaults_handling` test? Asking cause I'm not really familiar with that test.

Reply:
as long as `disable_web_page_preview` needs defaults handling, it should be covered in some way. But I grant you that `check_defaults_handling` is a mess 🙈